src/frontends/onnx/frontend/src/op/relu_6_13_14.hpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/core/deprecated.hpp"
+OPENVINO_SUPPRESS_DEPRECATED_START
```

Comment:
Should the new implementation be deprecated?

Reply:
until it contains ngraph - better to leave, easier for me to cleanup later